tools/webdriver/webdriver/bidi/modules/browsing_context.py

```diff
@@ -110,6 +111,38 @@ def handle_user_prompt(self,
         if user_text is not None:
             params["userText"] = user_text
         return params
+    
```

Comment:
I just also noticed that unittest job is failing because here and on the line `136` there are whitespaces.

Reply:
Hmm... `./wpt lint` doesn't find that. That kinda sucks. Will fix.